corehq/motech/openmrs/repeaters.py

```diff
@@ -48,6 +48,8 @@ class Meta(object):
     atom_feed_enabled = BooleanProperty(default=False)
     patients_last_polled_at = DateTimeProperty(default=None)
     patients_last_page = StringProperty(default=None)
+    encounters_last_polled_at = DateTimeProperty(default=None)
+    encounters_last_page = StringProperty(default=None)
```

Comment:
looking at how you've implemented other pieces did you consider making this a dict property:
```python
class AtomFeedStatus(DocumentSchema):
    last_polled_at = DateTimeProperty(default=None)
    last_page = StringProperty(default=None)

atom_feed_status = SchemaDictProperty(AtomFeedStatus)
```
That dict would be keyed by the feed name.

Reply:
Nice. That is much cleaner.